src/graphic/Text.ts

```diff
@@ -606,6 +606,10 @@ class ZRText extends Displayable<TextProps> {
         let xLeft = adjustTextX(baseX, contentBlock.contentWidth, textAlign);
         let lineTop = adjustTextY(baseY, contentBlock.contentHeight, verticalAlign);
 
+        if (textPadding) {
+            xLeft += textPadding[3];
```

Comment:
Should we consider `textAlign` before using adding? For example, if `textAlign` is `'center'`, maybe `xLeft` should not add the text padding?

Reply:
Yes, you are right, and also consider that `textAlign` was `right`, `xRight` should subtract padding @Ovilia

Reply:
It seems I made it wrong when refracting. We can roll it back to

https://github.com/ecomfe/zrender/blob/master/src/graphic/helper/text.js

```ts

        const boxX = adjustTextX(baseX, outerWidth, textAlign);
        const boxY = adjustTextY(baseY, outerHeight, verticalAlign);
        let xLeft = boxX;
        let lineTop = boxY;

        if (textPadding) {
            xLeft += textPadding[3];
        }

        let xRight = xLeft + innerWidth;

```

Reply:
@pissang done, I have revert the code